mousestyles/behavior/metrics.py

```diff
@@ -0,0 +1,153 @@
+from mousestyles import data
+
+from mousestyles import data
+from mousestyles import intervals
+
+
+def total_time(strain, mouse, day):
+    """Returns the total amount of time recorded for a certain mouse-day.
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+
+    Returns
+    -------
+    The total amount of time in seconds of the specified mouse-day.
+    """
+    start_seconds, end_seconds = data.load_start_time_end_time(
+        strain, mouse, day)
+    return end_seconds - start_seconds
+
+
+def active_time(strain, mouse, day):
+    """Returns the total amount of active time recorded for a certain mouse-day.
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+
+    Returns
+    -------
+    The total amount of active time in seconds for the specified mouse-day.
+    """
+    return create_intervals('AS', strain, mouse, day).measure()
+
+
+def total_amount(strain, mouse, day, feature):
+    """Returns the total amount consumed/moved for one of the following features:
+     food ("F"), water ("W"), and locomotion ("L").
+
+    Parameters
+    ----------
+    strain : int
+    mouse : int
+    day : int
+    feature : {'F', 'W', 'L'}
+        The feature used. 'F' for food, 'W' for water, 'L' for locomotion.
+
+    Returns
+    -------
+    The total amount consumed/moved for the given feature in the mouse-day.
+    """
+    df = data.load_all_features()
+    strain_cond = df['strain'] == strain
+    mouse_cond = df['mouse'] == mouse
+    day_cond = df['day'] == day
+
```

Comment:
These 3 lines can be deleted:

> ```
> strain_cond = df['strain'] == strain
> mouse_cond = df['mouse'] == mouse
> day_cond = df['day'] == day
> ```

We can put it in the next line. Don't feel it loses any clairity and creating tuples as separate variables does not seem necessary.

Otherwise if we do it this way, I can change the equivalent code in the `create_intervals` function. @dylandaniels please let me know which way you would prefer.